lib/content-services/src/lib/search/components/search-chip-autocomplete-input/search-chip-autocomplete-input.component.html

```diff
@@ -21,9 +27,14 @@
             (matChipInputTokenEnd)="add($event)">
     </mat-chip-list>
     <mat-autocomplete #auto="matAutocomplete" (optionSelected)="selected($event)" id="adf-search-chip-autocomplete">
-        <mat-option [disabled]="option | adfIsIncluded: selectedOptions" *ngFor="let option of filteredOptions$ | async"
-                    [ngClass]="(option | adfIsIncluded: selectedOptions) && 'adf-autocomplete-added-option'">
-            {{option}}
-        </mat-option>
+        <ng-container *ngIf="optionInput.value.length > 0">
+            <mat-option
+                [disabled]="option.id ? (option.id | adfIsIncluded: selectedOptions: 'id') : (option.value | adfIsIncluded: selectedOptions: 'value')"
+                *ngFor="let option of filteredOptions" [value]="option" matTooltipPosition="right" [matTooltipShowDelay]="tooltipShowDelay"
+                [matTooltip]="'SEARCH.RESULTS.WILL_CONTAIN' | translate:{searchTerm: option.fullPath ? option.fullPath : option.value}"
+                [ngClass]="(option.id ? (option.id | adfIsIncluded: selectedOptions: 'id') : (option.value | adfIsIncluded: selectedOptions: 'value')) && 'adf-autocomplete-added-option'">
```

Comment:
Again - should not this be configurable? What if anyone would need this for different field instead of id?